5-AccessControl/1-call-api-roles/SPA/src/pages/TodoList.jsx

```diff
@@ -44,11 +37,11 @@ export const TodoList = () => {
     };
 
     return (
-        <MsalAuthenticationTemplate 
-            interactionType={InteractionType.Redirect} 
+        <MsalAuthenticationTemplate
```

Comment:
The MsalAuthenticationTemplate doesn't seem to work. If I'm not logged in and I navigate to localhost:3000/dashboard, it should attempt to sign me in, but that's not happening. It could be that the RoleGuard is kicking in first, seeing there is no id token with the roles claim, prevents accessing the route, so that MsalAuthenticationTemplate never runs. Can you double check this?